pkg/api/v1beta3/dynakube/log_module_types.go

```diff
@@ -2,48 +2,48 @@ package dynakube
 
 import corev1 "k8s.io/api/core/v1"
 
-type LogMonitoringSpec struct {
+type LogModuleSpec struct {
 	Enabled bool `json:"enabled"`
 }
 
-type LogAgentSpec struct {
-	// Add custom LogAgent annotations
+type LogModuleDaemonSetSpec struct {
+	// Add custom annotations to the LogModule DaemonSet
```

Comment:
don't we only set it to pods? I'd just replace `LogModule Daemonset` with `Log Module` in this file

Reply:
should we still have `LogModule pods` ?

example: `// Set tolerations for the LogModule pods`

I would replace everything to `... LogModule pods`, WDYT?

Reply:
Updated, pls check 🙈